src/InputWithLabel.js

```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useEffect } from "react";
+
+
+function InputWithLabel({ todoTitle, handleTitleChange, children }) {
```

Comment:
Lesson instructions suggest using `props` as the function argument. Of course this is fine, but I think it's worth mentioning.